solvebio/cli/data.py

```diff
@@ -97,6 +98,43 @@ def should_exclude(path, exclude_paths, dry_run=False, print_logs=True):
     return False
 
 
+def _check_uploaded_folders(base_remote_path, local_start, all_folders):
```

Comment:
add **kwargs,

set `_client = ...` pass it through validate_full_path(...) and GlobalSearch(client=_client)

Reply:
As a general pattern, the idea is that `solvebio.cli` code shouldn't be run outside the CLI context. Anything that we move outside of here (e.g. _create_folder into `Object.create_folder`) will be set to accept `client`.

One of the goals of this ticket is to refactor `create_folders` to incorporate more of this logic: https://precisionformedicine.atlassian.net/browse/SBP-186?atlOrigin=eyJpIjoiZTY3ZmMwZjJjMTlhNGMzMzk5NTU1OTk4ZDRlMDVjYmEiLCJwIjoiaiJ9 so it will be done there.